VOKMockUrlProtocol.podspec

```diff
@@ -13,6 +13,6 @@ Pod::Spec.new do |s|
   s.requires_arc = true
 
   s.source_files = 'VOKMockUrlProtocol.[hm]'
-  s.dependency 'ILGHttpConstants', '~> 1.0.0'
+  s.dependency 'ILGHttpConstants', '~> 2.0.0'
```

Comment:
I figured I'd leave it at the same level as before - I feel like there's not going to be a huge need for people to have wildly divergent values for this.